cpp/data/pipeline_script_triggers

```diff
@@ -0,0 +1 @@
+/usr/local/var/tmp/ IN_CREATE,IN_DELETE /usr/local/bin/trigger_pipeline_script.sh $@$# $%
```

Comment:
Any reason why this is indented?

Reply:
This seems to be a display problem. There is no indentation in the file.